pkg/assets/builder.go

```diff
@@ -43,8 +45,10 @@ type Asset struct {
 	CanonicalLocation string
 }
 
-func NewAssetBuilder() *AssetBuilder {
-	return &AssetBuilder{}
+func NewAssetBuilder(assetRemapping *kops.Assets) *AssetBuilder {
```

Comment:
Depending on the sequencing of merges here, we will likely want to take a cluster as input... but we'll cross that bridge when we come to it